test/mithril.trust.js

```diff
@@ -64,5 +64,18 @@ describe("m.trust()", function () {
 			expect(root.innerHTML).to.equal("<div><p>&amp;copy;</p><p>©</p>©</div>")
 		})
 
+		// not sure that this goes here; not really an m.trust issue
```

Comment:
My bad...it goes in `mithril.render.js`. The tests should really be recategorized, but I just don't have the time to invest in fixing that.